lib/parse.js

```diff
@@ -21,6 +22,7 @@ function parse(css, options) {
     var needInfo = options && options.needInfo || Defaults.NEED_INFO;
     var rule = options && options.rule ||
         (syntax === 'js' ? Defaults.JS_RULE : Defaults.CSS_RULE);
+    var tabSize = options && options.tabSize || Defaults.TAB_SIZE;
```

Comment:
Agree. I will check `tabSize` is a number greater than 0.
I don't think it make sense to set a tabSize equal to 0.